frontend/src/containers/AdminHome.tsx

```diff
@@ -154,10 +147,19 @@ function AdminHome() {
           <p className="font-sans-md">
             {t("PDoASite")} <br /> {t("WantToViewPublishedSite")}
           </p>
-          <Button type="button" variant="outline" onClick={onViewPublicWebsite}>
-            {t("ViewPublishedSite")}{" "}
-            <FontAwesomeIcon size="sm" icon={faExternalLinkAlt} />
-          </Button>
+          <a
+            href="/"
+            target="_blank"
+            title="(opens in a new tab)"
+            className="link"
```

Comment:
Why "link" and not "usa-link"?

Reply:
I checked "Link.tsx" to see what class it was using and went off of that, I can change to "usa-link" if that's the preferred class for links.

Reply:
yes, pls. .link was created for table action and the main different is they keep color constant for all the states (hover, active, visited)